crates/driver/src/infra/config/file/load.rs

```diff
@@ -66,9 +66,23 @@ pub async fn load(network: &blockchain::Network, path: &Path) -> infra::Config {
                     solver::Liquidity::Fetch
                 },
                 account,
-                http_time_buffer: chrono::Duration::milliseconds(
-                    config.http_time_buffer_miliseconds.try_into().unwrap(),
-                ),
+                timeouts: solver::Timeouts {
+                    http_delay: chrono::Duration::milliseconds(
+                        config.http_time_buffer_milliseconds.try_into().unwrap(),
+                    ),
+                    solve_competition_time: chrono::Duration::milliseconds(
+                        config
+                            .solve_competition_time_buffer_milliseconds
+                            .try_into()
+                            .unwrap(),
+                    ),
+                    quote_competition_time: chrono::Duration::milliseconds(
+                        config
+                            .quote_competition_time_buffer_milliseconds
+                            .try_into()
+                            .unwrap(),
```

Comment:
Bit surprising that we are configuring hard coded values here if the complete solving/quoting time might change suddenly.
I think we should probably only have a single value for the http timeout and also have a approach where we simulate solutions as they get produced instead of simulating a bunch at the end of `/solve`.

Reply:
No sure I follow.
If `autopilot` gives X seconds to `driver` to execute how would you split it?
You can either split them this way, or say, 80% to solver and 20% to driver. Percents are maybe better to avoid having to restart driver with different configuration for these parameters whenever autopilot changes X.

> approach where we simulate solutions as they get produced

are you referring to the merges settlements or?

Reply:
Having relative time buffers would be one option. Imagine we are moving to 1 block per auction where we have 15s in total but might want to reserve some time for the autopilot. In that case there might be almost no time left.
Initially I implemented something like: `call "/solver" over and over until the time runs out`. In that setup it was more natural to make the most out of the time limit because you didn't have to commit to any time shares upfront.
Overall I think it might make sense to consider using websockets to stream solutions from the `solver` to the driver as they get computed. This would allow us to implement the more natural (IMO) streaming approach without having to manage additional state when calling `/solve` over and over again.

Reply:
Yes, that's a complete other story, in that case the deadline would be the same for both `driver` and `solvers` but it's not realistic to expect to have this any time soon I suppose.

Reply:
Switched to percents.